src/card.tsx

```diff
@@ -291,7 +292,7 @@ const useVideoWithControls = (
             playing: CardActionPauseIcon,
             loading: CardActionPauseIcon,
             paused: CardActionPlayIcon,
-            loadingTimeout: CardActionSpinner,
+            loadingTimeout: isRunningAcceptanceTest() ? CardActionPauseIcon : CardActionSpinner,
```

Comment:
Unrelated to this PR. This change fixes unstable screenshot tests in DisplayMedia/Poster cards with videos.